### PR TITLE
Add zoo-info-viewer --print-acls usage, document expected ZooKeeper ACLs

### PR DESCRIPTION
**Pull request by EdColeman**
- Add info for zoo-info-viewer --print-acls with output sample
- Updates ZooKeeper troubleshooting to include expected ACLs for Accumulo

### CODE REVIEW

**Comment by dlmarion** [comment]
Are you intending on merging this after the 2.1.1 release?

**Comment by EdColeman** [comment]
I thought it could be merged now - if someone builds 2.1.1 or from main the utility is there - but I'm not really clear how we handle doc updates and releases.

File: _docs-2/troubleshooting/zookeeper.md [hunk 1]
```diff
@@ -3,6 +3,34 @@ title: ZooKeeper
 category: troubleshooting
 order: 7
 ---
+## ZooKeeper ACLs
+
+Accumulo requires full access to nodes in ZooKeeper under the /accumulo path.  The ACLs can be examined using the
+ZooKeeper cli `getAcl` and modified with `setAcl` commands.  With 2.1.1, the zoo-info-viewer utility has an option
+that will print all of the ACLs for the nodes under `/accumulo/[INSTANCE_ID]` (See the [zoo-info-viewer]).
```
Comment (dlmarion) [inline _docs-2/troubleshooting/zookeeper.md:10]:
```suggestion
that will print all of the ACLs for the nodes under `/accumulo/[INSTANCE_ID]` (See [zoo-info-viewer]).  
```

File: _docs-2/troubleshooting/zookeeper.md [hunk 2]
```diff
@@ -3,6 +3,34 @@ title: ZooKeeper
...
 ---
+## ZooKeeper ACLs
+
+Accumulo requires full access to nodes in ZooKeeper under the /accumulo path.  The ACLs can be examined using the
+ZooKeeper cli `getAcl` and modified with `setAcl` commands.  With 2.1.1, the zoo-info-viewer utility has an option
+that will print all of the ACLs for the nodes under `/accumulo/[INSTANCE_ID]` (See the [zoo-info-viewer]).  
+To run the utility, only ZooKeeper needs to be running. If hdfs is running, the instance id can be read from hdfs, 
+or it can be entered with the zoo-info-viewer command --instanceId option.  Accumulo management processes 
+*do not* need to be running. This allows checking the ACLs before starting an upgrade. 
+
```
Comment (dlmarion) [inline _docs-2/troubleshooting/zookeeper.md:14]: Might be useful to include a one or two line example output.

File: _docs-2/troubleshooting/zookeeper.md [hunk 3]
```diff
@@ -3,6 +3,34 @@ title: ZooKeeper
...
+The utility prints out two summarized fields related to ZooKeeper ACL permissions:
```
Comment (dlmarion) [inline _docs-2/troubleshooting/zookeeper.md:15]:
```suggestion
The utility prints out a line for each znode that contains two fields related to ZooKeeper ACL permissions:
```

File: _docs-2/troubleshooting/zookeeper.md [hunk 4]
```diff
@@ -3,6 +3,34 @@ title: ZooKeeper
...
+   - `[ACCUMULO_OKAY | ERROR_ACCUMULO_MISSING_SOME]` - Are the permissions sufficient for Accumulo to operate 
+   - `[PRIVATE | NOT_PRIVATE]` - Can other users can read data from the ZooKeeper nodes.
+
+If there are nodes with `ERROR_ACCUMULO_MISSING_SOME` are nodes where Accumulo does not have `cdrwa` permissions.
```
Comment (dlmarion) [inline _docs-2/troubleshooting/zookeeper.md:19]:
```suggestion
Nodes marked with `ERROR_ACCUMULO_MISSING_SOME` means that Accumulo does not have `cdrwa` permissions.
```

File: _docs-2/troubleshooting/zookeeper.md (hunk 2 above)
Comment (EdColeman) [reply _docs-2/troubleshooting/zookeeper.md:14]: Added in 12938808

**Review by dlmarion — APPROVED** [review]

File: _docs-2/troubleshooting/tools.md [hunk 5]
```diff
@@ -396,3 +397,39 @@ table.iterator.majc.vers=20,org.apache.accumulo.core.iterators.user.VersioningIt
 ...
 -----------------------------------------------
 ```
+
+## mode: print ACLs
```
Comment (ctubbsii) [inline _docs-2/troubleshooting/tools.md:401]: Following the above format, could emphasize the version it was added in the header

```suggestion
## mode: print ACLs (new in 2.1.1)
```

**Comment by ctubbsii** [comment]
It's clearly marked as new in 2.1.1 now, so I think it's okay to merge now, so it will be there when we release.

**Review by ctubbsii — APPROVED** [review]

**Pull request merged**